src/clr-core/styles/layout/_type-vertical.scss

```diff
@@ -23,64 +23,57 @@
   display: inline-flex;
   flex-direction: column;
   justify-content: flex-start;
-  justify-items: flex-start;
   align-items: flex-start;
 
   & > *:last-child {
     margin-bottom: 0;
   }
 
   // Align Items
-  &[cds-layout~='align-items:top'] {
+  &[cds-layout~='align:top'] {
     justify-content: flex-start;
-    justify-items: flex-start;
   }
 
-  &[cds-layout~='align-items:bottom'] {
+  &[cds-layout~='align:bottom'] {
     justify-content: flex-end;
-    justify-items: flex-end;
   }
 
-  &[cds-layout~='align-items:left'] {
+  &[cds-layout~='align:left'] {
     align-items: flex-start;
   }
 
-  &[cds-layout~='align-items:right'] {
+  &[cds-layout~='align:right'] {
     align-items: flex-end;
   }
 
-  &[cds-layout~='align-items:vertical-center'] {
+  &[cds-layout~='align:vertical-center'] {
     justify-content: center;
-    justify-items: center;
```

Comment:
`justify-items` is a Flexbox property in the spec but currently ignored and is reserved for future Flexbox updates.